chatbot-core/backend/app/models/prompt.py

```diff
@@ -26,8 +32,17 @@ class Prompt(Base):
     id: Mapped[UNIQUEIDENTIFIER] = mapped_column(
         UNIQUEIDENTIFIER(as_uuid=True), primary_key=True, default=uuid4
     )
+    agent_id: Mapped[UNIQUEIDENTIFIER] = mapped_column(ForeignKey("agent.id"), nullable=False)
```

Comment:
Can you explain why we need field `agent_id` here?
Since `agent` and `prompt` are many-to-many. Why don't we implement another table to solve that?

Table: `agent_prompt`
- `agent_id`
- `prompt_id`

Reply:
I think we don't treat the prompt as the independent entity. Every agent has one prompt and vice versa. So this is the one-to-one relationship.

In other LLM applications like Poe, they allow us to create a Bot (similar to Agent). Each Bot has its own prompt and only that prompt can be used with that Bot and the Bot is used for a specific task (based on its prompt).

Reply:
Ok